mergin/client.py

```diff
@@ -1026,6 +1026,14 @@ def _download_file(self, project_path, file, directory, parallel=True, diff_only
         file_dir = os.path.dirname(os.path.normpath(os.path.join(directory, file['path'])))
         basename = os.path.basename(file['diff']['path']) if diff_only else os.path.basename(file['path'])
 
+        if file['size'] == 0:
+            directory = os.path.abspath(file_dir)
+            os.makedirs(directory, exist_ok=True)
+            print(os.path.join(file_dir, basename))
+            with open(os.path.join(file_dir, basename), 'w'):
```

Comment:
usually `with` is good approach, but here in the simple case I would do just `open(os.path.join(file_dir, basename), 'w').close()`